uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/daemon/http/HTTPSupport.java

```diff
@@ -28,7 +28,9 @@ public void contextInitialized(ServletContextEvent sce) {
         servletContext = sce.getServletContext();
         final JGitFileSystemProvider fsProvider = resolveProvider();
         if (fsProvider != null && fsProvider.getConfig().isHttpEnabled()) {
```

Comment:
```suggestion
        if (fsProvider != null) {
```
Shouldn't be this removed?